templates/resources.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import Adapt from 'core/js/adapt';
+import device from 'core/js/device';
+import { classes, templates } from 'core/js/reactHelpers';
+
+export default function Resources (props) {
+  const {
+    model,
+    resources,
+    onFilterClicked
```

Comment:
@swashbuck I was thinking that the definition of `onFilterClicked` would move here and you'd [use state](https://reactjs.org/docs/hooks-state.html) to keep track of what was currently selected - then you can remove those DOM manipulations (adding/removing classes, attributes etc) and let React handle that.